calico-enterprise/networking/gateway-api.mdx

```diff
@@ -0,0 +1,325 @@
+---
+description: Enable support for the Kubernetes Gateway API.
+---
+
+# Gateway API
+
+## Big picture
+
+Enable support for the Kubernetes Gateway API.
+
+## Value
+
+{{prodname}} includes support for the Kubernetes Gateway API, which allows advanced routing to services in a cluster, including weighted or blue-green load balancing.
+
+## Concepts
+
+### Gateway API
+
+The Gateway API is an official Kubernetes API for advanced routing to services in a cluster.  To read about its use cases, structure and design, please see [the official docs](https://gateway-api.sigs.k8s.io/).
```

Comment:
Should we specify which (minimum) gateway api version(s) we  support? I imagine that there may be conflicts with api versions, for example the grant is at v1beta1 now.

Reply:
Thanks, I've added a table to show this.